src/renderer/react/shapes/form.js

```diff
@@ -0,0 +1,8 @@
+import PropTypes from 'prop-types';
+
+// eslint-disable-next-line import/prefer-default-export
```

Comment:
I think we should disable this rule directly in .eslintrc.

Reply:
I think too

Reply:
+1